compositions/CollectionMenu/CollectionMenu.tsx

```diff
@@ -61,6 +61,7 @@ export function CollectionMenu() {
             paddingRight: 16,
```

Comment:
Is this in a style attr because it's overriding the variant?

If not, could do px='x4' as an atom

Reply:
this is an issue with button element padding - have to override in style block or with className (atoms don't work)
although removed because wasn't needed.